project_3/src/data.py

```diff
@@ -71,13 +71,14 @@ def __getitem__(self, idx):
         if fname is None:
             return []
 
-        img = self.img_transform(Image.open(os.path.join(self.img_path, fname)))
+        img = Image.open(os.path.join(self.img_path, fname)).convert('RGB')
+        img = self.img_transform(img)
         return img
 
 
 def pad_dataset(dataset, length):
     while len(dataset.fnames) < length:
-        dataset.fnames.append(None)
+        dataset.fnames.append(torch.empty(0))
```

Comment:
```suggestion
        dataset.fnames.append(None)
```
that's not where the data is outputed, it should be in line 72 I guess